server/config.py

```diff
@@ -40,6 +40,7 @@
 
 NEWBIE_BASE_MEAN = int(os.getenv('NEWBIE_BASE_MEAN', 500))
 NEWBIE_MIN_GAMES = int(os.getenv('NEWBIE_MIN_GAMES', 10))
+TOP_PLAYER_MIN_RATING = int(os.getenv('TOP_PLAYER_MIN_RATING', 2000))
```

Comment:
Talked to FTXCommando about this for a while, we're thinking 1600 is probably better.

Reply:
Sure!